services/director-v2/src/simcore_service_director_v2/modules/dynamic_sidecar/volumes_resolver.py

```diff
@@ -12,18 +72,51 @@ def target(cls, path: Path) -> str:
         target_path = cls.BASE_PATH / path.relative_to("/")
         return f"{target_path}"
 
+    @classmethod
+    def _volume_name(cls, path: Path) -> str:
+        return f"{path}".replace(os.sep, "_")
+
     @classmethod
     def _source(cls, compose_namespace: str, path: Path) -> str:
-        volume_name = f"{path}".replace(os.sep, "_")
-        return f"{compose_namespace}{volume_name}"
+        return f"{compose_namespace}{cls._volume_name(path)}"
 
     @classmethod
     def mount_entry(
-        cls, compose_namespace: str, path: Path, node_uuid: str
+        cls, compose_namespace: str, path: Path, node_uuid: NodeID
     ) -> Dict[str, Any]:
+        """
+        mounts local directories form the host where the service
+        dynamic-sidecar) is running.
+        """
         return {
             "Source": cls._source(compose_namespace, path),
             "Target": cls.target(path),
             "Type": "volume",
             "VolumeOptions": {"Labels": {"uuid": f"{node_uuid}"}},
         }
+
+    @classmethod
+    def mount_r_clone(
+        cls,
+        compose_namespace: str,
+        path: Path,
+        project_id: ProjectID,
+        node_uuid: NodeID,
+        r_clone_settings: RCloneSettings,
+    ) -> Dict[str, Any]:
+        return {
+            "Source": cls._source(compose_namespace, path),
+            "Target": cls.target(path),
+            "Type": "volume",
+            "VolumeOptions": {
+                "Labels": {
+                    "uuid": f"{node_uuid}",
```

Comment:
```suggestion
                    "node_uuid": f"{node_uuid}",
```

Reply:
I'd like to avoid breaking the garbage collection. If you want this I might need to do it in a separate PR.

Reply:
not sure I understand the reason why it would break the garbage collection. This is a volume, and I don't think this is trivial that the uuid label attached to it means it's a node.